src/Ajax.php

```diff
@@ -370,7 +370,7 @@ public static function createTabs(
                 $.get(
                   '{$CFG_GLPI['root_doc']}/ajax/updatecurrenttab.php',
                   {
-                     itemtype: '" . addslashes($type) . "',
+                     itemtype: " . json_encode($type) . ",
```

Comment:
I'm not sure I understand this one.

Reply:
`addslashes` was probably doing the job here, but the best way to print a safe JS value is to use `json_encode`.